main.go

```diff
@@ -205,7 +205,7 @@ func serve() {
 	e.File("/favicon.png", "favicon.png")
 
 	// TODO: can use more caching here
-	e.Group(fmt.Sprintf("/%s/static/", pathPrefix), gzip, middleware.Static("templates/static/dist/"))
+	e.Group(fmt.Sprintf("/%s/static/", pathPrefix), gzip, middleware.Static("handlers/templates/static/dist/"))
```

Comment:
This is going to be replaced by a `http.FileServer(handlers.Assets)` when #48 is done.